realm/src/androidTest/java/io/realm/services/InterprocessService.java

```diff
@@ -0,0 +1,130 @@
+package io.realm.services;
+
+import android.app.Service;
+import android.content.Intent;
+import android.os.Bundle;
+import android.os.Handler;
+import android.os.IBinder;
+import android.os.Message;
+import android.os.Messenger;
+import android.os.RemoteException;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import io.realm.Realm;
+import io.realm.entities.AllTypes;
+
+/**
+ * Helper service for multi-processes support testing.
+ */
+public class InterprocessService extends Service {
```

Comment:
Perhaps rename this to RemoteProcessService? that indicates better that it is the "other" process and not the "main" one IMO